aptos-move/framework/aptos-experimental/sources/confidential_asset/confidential_proof.move

```diff
@@ -1056,6 +1057,15 @@ module aptos_experimental::confidential_proof {
         )
     }
 
+    native fun verify_batch_range_proof_internal(
```

Comment:
Ah!

Reply:
And you would just map this to the same Rust function, right?

Reply:
And we would never enable batch bulletproofs in `ristretto255_bulletproofs`?

Reply:
we can also "always abort" there, and reuse the feature flag for protecting this new native.